pylint/lint/pylinter.py

```diff
@@ -495,7 +497,7 @@ def _parse_error_mode(self) -> None:
         self.set_option("persistent", False)
         self.set_option("score", False)
 
-    def list_messages_enabled(self):
+    def list_messages_enabled(self) -> None:
```

Comment:
No related to the MR, but I hate those function that mix creating a string and displaying. ``print`` is duplicated everywhere, it's just a mess.